src/components/atoms/select/Select.tsx

```diff
@@ -0,0 +1,51 @@
+import { RefObject } from "react";
+import { Select as BaseSelect } from "@base-ui-components/react";
+import { LuChevronsUpDown } from "react-icons/lu";
+
+interface SelectProps {
+  defaultValue?: string;
+  value?: string;
+  options?: string[];
```

Comment:
Thought: in the future we'll likely need to make `options` an object with a display text and separate value, but for now this is A-OK.